### PR TITLE
[32822] Fix parsing budget with non-english locale

### PR DESCRIPTION
**Pull request by oliverguenther**
https://community.openproject.com/wp/32822

### CODE REVIEW

File: modules/costs/app/views/cost_objects/items/_material_budget_item.html.erb [hunk 1]
```diff
@@ -99,5 +99,5 @@ See docs/COPYRIGHT.rdoc for more details.
       <span class="hidden-for-sighted"><%= t(:button_delete) %></span>
     </a>
   </td>
-</tr>
+< tr>
```
Comment (akasparas) [inline modules/costs/app/views/cost_objects/items/_material_budget_item.html.erb]: This change looks like mistake.

Comment (oliverguenther) [reply modules/costs/app/views/cost_objects/items/_material_budget_item.html.erb]: Good catch, thanks @akasparas :bowing_man:

**Comment by ulferts** [comment]
Without having looked into the why, the error still occurs on my machine.

**Comment by oliverguenther** [comment]
Still works correctly with english and german locale for me on FF/Chrome. Could you please elaborate?

**Comment by ulferts** [comment]
Took me a while but it only seems to happen for values, that do not actually need a decimal separator, e.g. 10,00. This is then transmitted as 10.0 to the server which then strips away the decimal separator. I think the last operation is no longer necessary and might be omitted completely.

**Comment by ulferts** [comment]
@oliverguenther, I believe the special case should also be handled now.

**Comment by oliverguenther** [comment]
LGTM, thanks @ulferts :+1:

**Pull request merged**